manifests/init.pp

```diff
@@ -160,6 +160,8 @@
 #
 # $cors_domains::                 List of domains that show be allowed for Cross-Origin Resource Sharing
 #
+# $trusted_proxies::              List of trusted HTTP(s) reverse proxies, which are allowed to access.
```

Comment:
To access... what? I suppose the implication is the Foreman puma application? But I would opt to tweak the wording to make this clearer.

Reply:
which are allowed to access the API?